wallet/tutorials/javascript-dapp-simple.md

```diff
@@ -0,0 +1,372 @@
+---
+description: Create a simple dapp to integrate with MetaMask.
+toc_max_heading_level: 4
+sidebar_position: 3
+---
+
+# Create a simple dapp 
+
+This tutorial demonstrates the basics of creating a simple dapp using JavaScript that integrates with MetaMask. 
+
+MetaMask allows dapps to handle transactions, authenticate users through Ethereum addresses instead of traditional logins, and engage with smart contracts directly.
+
+:::important Learning tutorial
+This tutorial is for educational purposes and connects to MetaMask using the legacy provider object, `window.ethereum`.
+For deployment in a production environment, we recommend [connecting to MetaMask using EIP-6963](../how-to/connect/index.md) instead. 
+
+[EIP-6963](https://eips.ethereum.org/EIPS/eip-6963) introduces an alternative wallet detection mechanism to the `window.ethereum` provider, and enables dapps to support [wallet interoperability](../concepts/wallet-interoperability.md).
+
+For a full end-to-end tutorial that can be used in production, see the
+[Create a simple React dapp](../tutorials/react-dapp-local-state.md) tutorial.
+:::
+
+## Prerequisites
+
+- [MetaMask](https://metamask.io/) [installed in the browser of your choice](/snaps/get-started/install-flask/) on your development machine.
+ We recommend [running a development network](/wallet/how-to/run-devnet/) on MetaMask when developing a dapp.
+
+- A text editor of your choice, such as [VS Code](https://code.visualstudio.com/).
+ You can install the [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
+extension for VS Code to easily launch a local development server for your dapp.
+
+- A module bundler, such as [Webpack](https://github.com/webpack/webpack).
+
```

Comment:
```suggestion
```

Since we instruct the user to install Webpack in one of the steps, we don't need it here as a prerequisite.